src/main/java/no/nav/familie/inntektsmelding/utils/FormatUtils.java

```diff
@@ -0,0 +1,49 @@
+package no.nav.familie.inntektsmelding.utils;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.TextStyle;
+import java.util.Locale;
+
+import static java.time.format.DateTimeFormatter.ofPattern;
+
+public final class FormatUtils {
```

Comment:
Kan endre til norsk navn. f.eks. 
```suggestion
public final class FormateringUtils {
```

Reply:
Format er norsk det og;)

Reply:
Hehe true, vi lar den være